test/fix-me.js

```diff
@@ -1,5 +1,14 @@
-var expect = require('chai').expect
-var fixMe = require('../lib/fix-me.js')
+var expect = require('chai').expect,
+    fs = require('fs'),
+    intercept = require("intercept-stdout"),
+    captured_text = "",
+    unhook_intercept = intercept(function(txt) {
```

Comment:
I'd subscribe to the idea that tests should be able to be read top down, even if it means duplication between them.